src/Storages/System/StorageSystemColumns.cpp

```diff
@@ -74,6 +74,7 @@ class ColumnsSource : public ISource
         : ISource(header_)
         , columns_mask(std::move(columns_mask_)), max_block_size(max_block_size_)
         , databases(std::move(databases_)), tables(std::move(tables_)), storages(std::move(storages_))
+        , clientInfo(context->getClientInfo())
```

Comment:
clientInfo -> client_info,
but it is better not to copy the whole struct here but only check for a particular member and set a flag.